cla_backend/apps/core/middleware.py

```diff
@@ -1,12 +1,3 @@
-from django.http import Http404
-from django_statsd.clients import statsd
-
-
 class GraphiteMiddleware(object):
```

Comment:
Can we remove the entire middleware as it seems like it's not doing anything but just returning the given response